bitrise.yml

```diff
@@ -293,10 +293,6 @@ workflows:
             - module: trace-sdk
             - arguments: "--stacktrace"
           title: android_build
-      - gradle-runner@1:
```

Comment:
not sure removing the unit tests here is the best idea - possibly just revert the previous workflow step and run the unit tests without coverage? I presume the unit tests then get run later with the coverage at the same time the instrumentation tests run?